spec/requests/api/opds/V2/feeds_spec.rb

```diff
@@ -125,6 +125,7 @@
 
         it 'is non-empty' do
           get api_opds_amherst_path, headers: headers
+          pp response.body
```

Comment:
Probably don't want the pp in here.

Reply:
Thanks bud. This spec being broken really had me stumped. It was eating the error and this pp really did the job!!
Will remove.